pkg/sdn/plugin/egress_dns.go

```diff
@@ -0,0 +1,137 @@
+package plugin
+
+import (
+	"net"
+	"sync"
+	"time"
+
+	"github.com/golang/glog"
+
+	osapi "github.com/openshift/origin/pkg/sdn/api"
+
+	ktypes "k8s.io/kubernetes/pkg/types"
+)
+
+const (
+	defaultEgressDNSTTLSyncDuration = 8 * time.Hour
```

Comment:
This gets defined here, but used from a different file in a different commit. I think you could just have the "syncEgressDNSTTL" goroutine here rather than in egress_network_policy.go though.